spec/menu_spec.rb

```diff
@@ -0,0 +1,8 @@
+require "Menu"
+describe Menu do
+  it "allow you to read dishes and prices" do
+    my_menu = double("Tapas", dish: "Pinchitos", price: 6.99)
+    expect(my_menu.dish).to eq("Pinchitos")
+    expect(my_menu.price).to eq(6.99)
```

Comment:
By replacing the menu object was a double you're affectively no longer testing the code in your menu class but instead of testing the double.

One way to realise you're doing something like this: delete the code in the class you're testing (in this case the `dish` and `price` methods in your `menu.rb`). If your tests still passes, it's an indication that your test isn't really testing what you want it to test.

You can testing my new class directly without a double:
```
 my_menu = Menu.new("Pinchitos")
 expect(my_menu.dish).to eq("Pinchitos")
``` 

We only want to use doubles to replace classes that interact with the class we are testing. It might be worth revisiting some of the materials on doubles and testing and chatting with a coach about this!